thirdparty/faiss/faiss/IndexIVFAdditiveQuantizer.cpp

```diff
@@ -212,6 +212,7 @@ struct AQInvertedListScannerLUT : AQInvertedListScanner {
         AQInvertedListScanner::set_query(query_vector);
         if (!is_IP && !ia.by_residual) {
             distance_bias = fvec_norm_L2sqr(query_vector, ia.d);
+            distance_bias = (distance_bias == 0.0 ? 1.0 : distance_bias);
```

Comment:
no, because it uses norm for distance = x^2 - 2xy + y^2 calculation, not for a cosine-related code